package.json

```diff
@@ -3,11 +3,13 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "dev": "npx nx serve poker-roll"
+    "dev": "nx run-many -t serve"
```

Comment:
Nx command to run many tasks at once: https://nx.dev/packages/nx/documents/run-many